src/profile-logic/marker-data.js

```diff
@@ -1056,6 +1082,13 @@ export function getMarkerFullDescription(marker: Marker) {
       case 'Text':
         description += ` — ${data.name}`;
         break;
+      case 'IPC':
+        description = `[${data.thispid}${data.direction ? '<-' : '->'}${
```

Comment:
Some comments showing examples of what this spits out would be helpful, the line right now is hard to visually parse.

Reply:
I'm planning on reworking this anyway, since the majority of the information here is debug info I was using to check that I'm correlating IPC markers correctly. If it's still complicated, I'll definitely add a comment though!